File: test/plugin/android/plugin/src/main/java/com/microsoft/c3p/test/TestEvent.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for details.

package com.microsoft.c3p.test;

import java.util.EventObject;

public class TestEvent extends EventObject {
    private int _counter;

    TestEvent(Object source, int counter) {
        super(source);
        _counter = counter;
    }

    public int getCounter() {
        return _counter;
    }
}

File: test/plugin/windows-cs/TestEvent.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for details.

using System;

namespace Microsoft.C3P.Test
{
    public sealed class TestEvent
    {
        internal TestEvent(int counter)
        {
            this.Counter = counter;
        }

        public int Counter { get; private set; }
    }
}
